packages/instant/src/containers/selected_asset_buy_order_state_buttons.ts

```diff
@@ -107,7 +108,26 @@ const mapDispatchToProps = (
     },
 });
 
-export const SelectedAssetBuyOrderStateButtons: React.ComponentClass<SelectedAssetBuyOrderStateButtons> = connect(
+const mergeProps = (
+    connectedState: ConnectedState,
+    connectedDispatch: ConnectedDispatch,
+    ownProps: SelectedAssetBuyOrderStateButtons,
```

Comment:
Is this the correct type for `ownProps`?

Reply:
So it looks like `SelectedAssetBuyOrderStateButtons` is being used as "props" for both `mapStateToProps` and `mapDispatchToProps`.

`SelectedAssetBuyOrderStateButtons` is defined as
```
export interface SelectedAssetBuyOrderStateButtons {}
```

Reply:
There also exists a suspicious `BuyOrderStateButtonProps` (without the `Selected` prefix) that contains a lot of properties. This interface is present in `buy_order_state_button.ts` and it's never imported/used within `selected_asset_buy_order_state_buttons.ts` so I assume it's different. Can you confirm?

Reply:
Oh that makes sense. All good.